### PR TITLE
Add new libs

### PR DESCRIPTION
**Pull request by khalifapro**
Signed-off-by: “khalifapro” <enkhalifapro@yahoo.com>

### CODE REVIEW

File: affiliation/dto.go [hunk 1]
```diff
@@ -0,0 +1,115 @@
+package affiliation
```
Comment (lukaszgryglicki) [inline affiliation/dto.go:1]: Why do we need affiliation? My understanding is that V2 is only outputting "identities" (not even profiles/users). V2 is not calling any affiliations APIs nor is calculating org name(s)?
not a stopper - just wondering.

File: affiliation/identity.go [hunk 2]
```diff
@@ -0,0 +1,437 @@
...
+		return a.GetIdentityByUser(key, value)
+	}
+
+	var ident IdentityData
+	err = json.Unmarshal(res, &ident)
+	if err != nil {
+		return nil, err
+	}
+
+	profileEndpoint := a.AffBaseURL + "/affiliation/" + url.PathEscape(a.ProjectSlug) + "/get_profile/" + *ident.UUID
```
Comment (lukaszgryglicki) [inline affiliation/identity.go:271]: V2 isn't supposed to call affiliations API IMHO.

File: auth0/dto.go [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+// Resp struct
+type Resp struct {
+	AccessToken string `json:"access_token"`
+	Scope       string `json:"scope"`
+	ExpiresIn   int    `json:"expires_in"`
+	TokenType   string `json:"token_type"`
+}
+
+// ESTokenSchema for AuthToken
+type ESTokenSchema struct {
```
Comment (lukaszgryglicki) [inline auth0/dto.go:21]: Same here with tokens - what do we need token for in V2?

Comment (enkhalifapro) [reply auth0/dto.go:21]: Yes, you are right, it's just temp will be removed

File: affiliation/identity.go (hunk 2 above)
Comment (enkhalifapro) [reply affiliation/identity.go:271]: Yes, you are right, it's just temp will be removed

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm - this is all just new stuff.
One note: why do we need any affiliations/auth0 related stuff in V2?
You can merge, no problem, but I just wonder...
cc @fayazg @1010sachin

**Comment by lukaszgryglicki** [comment]
So if this is temp and not needed then can you remove this before merging this PR @khalifapro ?

File: auth0/dto.go (hunk 3 above)
Comment (enkhalifapro) [reply auth0/dto.go:21]: let me remove it now

Comment (lukaszgryglicki) [reply auth0/dto.go:21]: Thanks! basically all auth0/profile related stuff.
I remember on standup - we agreed that those new DSes will output identities and that's it, so someting like SH DB `identities` table - but not `profiles` table.
Just identity data like: `(UUID, source, name, username, email)`.

**Comment by lukaszgryglicki** [comment]
I see you've removed non-needed stuff, I've already approved in the first round... so you can merge anytime.

**Pull request merged**